### PR TITLE
Align globe examples with gl js

### PR DESCRIPTION
**Pull request by kiryldz**
### Summary of changes

Following examples are aligned:
 - Basic globe
 - Spinning globe
 - Heatmap on a globe
 - Flyto

We don't have clear equivalent for 
 - Styling the atmosphere
 - Animate a point along route

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          color(rgb(220, 159, 159)) // Pink fog / lower atmosphere
+          highColor(rgb(220, 159, 159)) // Blue sky / upper atmosphere
+          horizonBlend(0.4) // Exaggerate atmosphere (default is .1)
+        }
+        +rasterDemSource("raster-dem") {
+          url("mapbox://mapbox.terrain-rgb")
+        }
+        +terrain("raster-dem") {
+          // camera seems to be a bit jumping on high zoom level - check with gl-native
+          exaggeration(1.5)
```
Comment (pengdev) [inline app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt:50]: nit: extract to constants.

**Review by pengdev — APPROVED** [review]
LGTM % nits.

File: app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt (hunk 1 above)
Comment (kiryldz) [reply app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt:50]: I'm keeping the code as close to gl-js as possible so will leave as is here 😉

File: app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.extension.style.terrain.generated.terrain
+import com.mapbox.maps.plugin.animation.MapAnimationOptions.Companion.mapAnimationOptions
+import com.mapbox.maps.plugin.animation.flyTo
+import com.mapbox.maps.plugin.gestures.OnMapClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapClickListener
+import com.mapbox.maps.testapp.R
+
+/**
+ * Use [MapboxMap.flyTo] on a map with globe projection to slowly zoom to a location.
```
[the remark this answers is not in the document]
Comment (kiryldz) [reply app/src/main/java/com/mapbox/maps/testapp/examples/globe/GlobeFlyToActivity.kt:25]: we actually plan to publish this example somewhere on www, it makes not that much sense to ref specifically to gl-js cause same example will be also for iOS. let's leave here as is and add refs if needed on www to similar examples

File: app/src/main/res/values/example_descriptions.xml [hunk 3]
```diff
@@ -84,7 +84,8 @@
     <string name="description_large_geojson">Display long route as large geojson</string>
     <string name="description_nine_patch">Add 9-patch image to the style</string>
     <string name="description_feature_state">Create interactive hover effects with feature state</string>
-    <string name="description_3d_globe">Showcasing 3D globe</string>
+    <string name="description_3d_globe">Create a map using the globe projection.</string>
+    <string name="description_spinning_globe">Display your map as an interactive, rotating globe.</string>
```
Comment (yunikkk) [inline app/src/main/res/values/example_descriptions.xml:88]: Also here - maybe align `display` / `create`

**Review by yunikkk — APPROVED** [review]

File: app/src/main/res/values/example_descriptions.xml (hunk 3 above)
Comment (kiryldz) [reply app/src/main/res/values/example_descriptions.xml:88]: was aligned with gl-js, not my texting 😄

**Pull request merged**